packages/plexus/src/LayoutManager/Coordinator.tsx

```diff
@@ -17,33 +15,41 @@
 /* eslint-disable no-console */
 
 import * as convCoord from './dot/conv-coord';
-import workerFactory from './layout.worker';
+import LayoutWorker from 'worker/LayoutManager/layout.worker';
```

Comment:
is this now from an npm package? if so I believe it should be the first import with a new line after.

Reply:
Well, I needed to create a type definition for the layout.worker that reflected it being used by `Coordinator.tsx` as a `WebWorker`. In TypeScript, I don't think you can create auxiliary type defs for local files. So, I made an alias in webpack and then added a type def for the alias. 

So, it's a local file loaded via an alias resolver.  

Seems like it should be kept in the local files section of imports, but moved to the bottom.